modules/unsupported/mbstyle/src/test/java/org/geotools/mbstyle/SymbolMBLayerTest.java

```diff
@@ -31,17 +36,20 @@
     SymbolMBLayer testLayerDefault;
     SymbolMBLayer testLayer;
     MBStyle defaultStyle;
-
+    MBStyle style;
+    List<FeatureTypeStyle> ftsd;
+    List<FeatureTypeStyle> fts;
```

Comment:
These variable names are a bit ambiguous, especially considering the types are identical